core/src/main/java/com/orange/ouds/core/theme/OudsTheme.kt

```diff
@@ -42,6 +42,9 @@ private val LocalSpaces = staticCompositionLocalOf<OudsSpaces> { missingComposit
 private val LocalComponentsTokens = staticCompositionLocalOf<OudsComponentsTokens> { missingCompositionLocalError("LocalComponentsTokens") }
 internal val LocalColoredBox = staticCompositionLocalOf<Boolean> { false }
 
+/**
+ * @suppress
+ */
 object OudsTheme {
```

Comment:
`OudsTheme` can be used to retrieve some useful tokens, so I think it should be documented.